docker_images/diffusers/app/pipelines/image_to_image.py

```diff
@@ -94,10 +96,10 @@ def __call__(self, image: Image.Image, inputs: str = "", **kwargs) -> "Image.Ima
             if "num_inference_steps" not in kwargs:
                 kwargs["num_inference_steps"] = 25
             images = self.ldm(
-                inputs,
-                image,
+                prompt,
+                image=inputs,
```

Comment:
This make me double `s/image/inputs` is necessary.

We could by all means keep the name `image` here no ? (Since the validation accepts both and sends them as a positional argument).

Works for me either way, but renaming in such way is always something I'm careful about.

Reply:
ow, I see what you mean, since the the position matches the image? that totally works ☺️ right?
`self.ldm(prompt, inputs, **kwargs)["images"]`

Reply:
or did you mean, not rename the `image` to `inputs`?

`def __call__(self, image: Image.Image, prompt: str = "", **kwargs) -> "Image.Image":`

Reply:
btw,, if you or @mishig25 would like to update/merge this based on your feedback tomorrow morning, please feel free to do so. We don't need to delay this, especially considering the few dependent issues 🙏

Reply:
I meant not reanme `image` to `inputs` (in this particular piece of code, the widget and the API should still handle both)

Reply:
ok @Narsil that makes sense! I've update the code